lab_2_keywords_cooccurrence/main.py

```diff
@@ -69,7 +153,12 @@ def calculate_word_scores(word_degrees: Mapping[str, int],
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if not check_dict(word_degrees, str, int, False) or not check_dict(word_frequencies, str, int, False):
+        return None
+    for word in word_degrees:
+        if word not in word_frequencies:
+            return None
+    return {word: word_degrees[word] / word_frequencies[word] for word in word_degrees if word in word_frequencies}
```

Comment:
you've already checked that all keys of `word_degrees` are present in `word_frequencies`